garden_ai/app/console.py

```diff
@@ -1,3 +1,114 @@
 from rich.console import Console
+from rich.table import Table
+
+from typing import List, Optional, Any
 
 console = Console()
+
+
+def _get_rich_resource_table(
```

Comment:
Since this function is doing the "heavy lifting" in this module, could we clean it up a bit? I think something like this should be equivalent, and simplifying makes it easier to pick out the moving parts at a glance 
```suggestion
def _get_rich_resource_table(
    resource_objs: Optional[List[Any]], 
    resource_table_cols: List[str], 
    table_name: str
) -> Table:
    
    table = Table(title=table_name)
    for col in resource_table_cols:
        table.add_column(col)
    
    for resource_obj in resource_objs:
        row = []
        for field in resource_table_cols:
            row.append(str(getattr(resource_obj, field)))
        table.add_row(*row)
    
    return table
```